redfish-core/lib/log_services.hpp

```diff
@@ -742,6 +797,14 @@ inline void
                     crow::utility::getDateTime(timestamp);
                 asyncResp->res.jsonValue["AdditionalDataSizeBytes"] = size;
 
+                if (sessionId != "")
+                {
+                    asyncResp->res.jsonValue["Oem"]["OpenBMC"]["@odata.type"] =
+                        "#OemLogEntry.v1_0_0.LogEntry";
+                    asyncResp->res.jsonValue["Oem"]["OpenBMC"]["InitiatedBy"] =
+                        "/redfish/v1/SessionService/Sessions/" + sessionId;
```

Comment:
Please return only sessionId as a string here. This is not a url

Reply:
Done